terraform_alert_config/import_adapter/README.md

```diff
@@ -0,0 +1,68 @@
+# Prepare workspace
```

Comment:
Nitpick: can we use smaller headings? (## rather than # for the headings) I think this would make the doc easier to read